pkg/agent/agent.go

```diff
@@ -198,78 +197,36 @@ func (f *Flows) interfacesManager(ctx context.Context) (<-chan *flow.Record, err
 }
 
 // processRecords creates the tracers --> accounter --> forwarder Flow processing graph
-func (f *Flows) processRecords(tracedRecords <-chan *flow.Record) *node.Terminal {
+func (f *Flows) processRecords(tracedRecords <-chan []*flow.Record) *node.Terminal {
 	// The start node receives Records from the eBPF flow tracers. Currently it is just an external
 	// channel forwarder, as the Pipes library does not yet accept
 	// adding/removing nodes dynamically: https://github.com/mariomac/pipes/issues/5
 	alog.Debug("registering tracers' input")
-	tracersCollector := node.AsInit(func(out chan<- *flow.Record) {
+	tracersCollector := node.AsInit(func(out chan<- []*flow.Record) {
 		for i := range tracedRecords {
 			out <- i
 		}
 	})
-	alog.Debug("registering accounter")
-	accounter := node.AsMiddle(f.accounter.Account)
 	alog.Debug("registering exporter")
 	export := node.AsTerminal(f.exporter)
 	alog.Debug("connecting graph")
-	tracersCollector.SendsTo(accounter)
-	accounter.SendsTo(export)
+	tracersCollector.SendsTo(export)
```

Comment:
The accounter is not anymore a part of the main pipeline, as we account flows in the kernel-side.
The accounter instance has been moved to the ringbuffer reader, which is the only part that still needs to aggregate flows at userspace.